lib/checks/050-koenig-css-classes.js

```diff
@@ -63,7 +63,7 @@ const checkKoenigCssClasses = function checkKoenigCssClasses(theme, options) {
             theme.results.fail[ruleCode] = {};
             theme.results.fail[ruleCode].failures = [
                 {
-                    ref: 'styles'
+                    ref: 'Missing styles or stylesheets'
```

Comment:
This message appears under "Affected files" section, so would read like:
```
Affected files:
- Missing styles or stylesheets
```

I think "Missing stylesheets with required class styles" would be a more suitable reference here.

Reply:
Agreed! Updated reference text